opensearch-operator/pkg/builders/cluster.go

```diff
@@ -185,6 +185,22 @@ func NewSTSForNodePool(
 
 	image := helpers.ResolveImage(cr, &node)
 
+	var mainCommand []string
+	var com string
```

Comment:
Declaration for `com` can be moved into the if as it is only used inside the if branch.
And it can be combined into one line: `com := "./bin/opensearch-plugin install --batch"`